bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/base/Instrument.java

```diff
@@ -17,8 +18,28 @@
  */
 public abstract class Instrument {
 
+    public enum Bands {
```

Comment:
This is more of a representation of a number of bands an instrument is covering or the instrument type, i.e. visible light, vs near IR vs far IR. I do wonder however if we could use something like that in other places (or already have something similar)?